Sources/LeafKit/String+HTMLEscape.swift

```diff
@@ -1,103 +1,25 @@
+#if swift(<6.0)
 import Foundation
```

Comment:
Maybe dance the `FoundationEssentials` dance wherever it's used?

Reply:
Then I'd start having to go changing all the `import NIO`s to `import NIOCore` and such too, that takes effort 😆

Reply:
There, I did both. Happy? 😁